tts/src/win.rs

```diff
@@ -9,10 +10,15 @@ pub struct WinConfig {
     pub voice: Option<String>,
 }
 
+static MUTEX: Mutex<()> = Mutex::const_new(());
+
 pub async fn synthesize(text: &str, config: &WinConfig) -> Result<Vec<Vec<u8>>, WinError> {
     // Note, there does not seem to be a way to explicitly set 16000kHz, 16 audio bits per
     // sample and mono channel.
 
+    // Prevent concurrent Windows TTS synthesis, as this might cause a crash.
+    let lock = MUTEX.lock().await;
```

Comment:
A semaphore with a reasonable concurrency limit might also do the trick - would would require some additional testing to find a good limit

Reply:
Lets stop the bleeding with this change and then we can consider setting a concurrency.

I am fine with this but the checks are failing.